app/config/cors.yml

```diff
@@ -0,0 +1,16 @@
+nelmio_cors:
+    defaults:
+        allow_credentials: false
+        allow_origin: []
+        allow_headers: []
+        allow_methods: []
+        expose_headers: []
+        max_age: 0
+        hosts: []
+        origin_regex: true
+        forced_allow_origin_value: ~
+    paths:
+        '^/api/v3/':
+            allow_origin: ['^https?://localhost:[0-9]+', 'https?:\/\/(\w+\.)?dismoi\.io']
```

Comment:
`https://sub.domain.dismoi.io` would not work … I suggest `https?:\/\/(\w+\.)*dismoi\.io`

Reply:
yy why not, done